website/docs/user-group-role/roles/manage_roles.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Manage roles"
+description: "Learn how to work with roles and permissions in authentik."
+---
+
+Roles are a collection of permissions, which can then be assigned, en masse, to a group. Using roles is a way to quickly grant permissions; by adding a user to the group with the appropriate assigned roles, any user in that group then inherits all of those permissions that are assigned to the role.
+
+:::info
+In authentik, we assign roles to groups, not to individual users.
+:::
+
+## Create a role
+
+To create a new role, follow these steps:
+
+1. In the Admin interface, navigate to **Directory > Roles**.
+2. Click **Create**, define the name of the role, and then click **Create** in the modal.
+3. Next, [assign permissions to the role](../access-control/permissions.md#assign-or-remove-permissions-for-a-specific-role).
+
+## Modify a role
+
+To modify a role, follow these steps:
+
+-   To edit the name of the role click the Edit icon beside the role's name.
```

Comment:
Going the other way, because the dependent clause comes _first_ and there is no preposition separating the two clauses, I think a comma _is_ needed here just as you have it on line 30 below.

"To edit the name of the role, click the Edit icon beside the role's name."

Otherwise, you end up with the hard-to-scan phrase "...the role click..." and have to re-read the sentence to parse it correcctly.

Reply:
Finally I get to use a comma!